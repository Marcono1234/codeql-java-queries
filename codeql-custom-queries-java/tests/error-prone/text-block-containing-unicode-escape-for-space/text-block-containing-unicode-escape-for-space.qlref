./error-prone/text-block-containing-unicode-escape-for-space.ql
