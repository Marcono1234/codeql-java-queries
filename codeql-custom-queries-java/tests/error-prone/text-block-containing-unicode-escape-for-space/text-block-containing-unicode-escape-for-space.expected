| Test.java:3:9:5:11 | """\n        \\u0020 \\u0009 \\u000c \\u000C\n        """ | Contains Unicode escape `\\u000C`; should use `\\f` |
| Test.java:3:9:5:11 | """\n        \\u0020 \\u0009 \\u000c \\u000C\n        """ | Contains Unicode escape `\\u000c`; should use `\\f` |
| Test.java:3:9:5:11 | """\n        \\u0020 \\u0009 \\u000c \\u000C\n        """ | Contains Unicode escape `\\u0009`; should use `\\t` |
| Test.java:3:9:5:11 | """\n        \\u0020 \\u0009 \\u000c \\u000C\n        """ | Contains Unicode escape `\\u0020`; should use `\\s` |
| Test.java:6:9:8:11 | """\n        \\uuuu0020\n        """ | Contains Unicode escape `\\uuuu0020`; should use `\\s` |
| Test.java:10:9:11:11 | """   \\u0020\n        """ | Contains Unicode escape `\\u0020`; should use `\\s` |
